### PR TITLE
Added ability to logging output from executable

### PR DESCRIPTION
**Pull request by cravler**

### CODE REVIEW

**Comment by ricbra** [comment]
Could you add a description what you added and why?

**Comment by cravler** [comment]
If using --output flag, then stdout from executable goes to info.log and stderr to error.log

**Comment by andrefigueira** [comment]
I'd love this, as atm, only thing i get out of supervisord tail is: 

2017/06/27 09:21:58 Processing message...
2017/06/27 09:21:58 Compressed message
2017/06/27 09:21:58 Processing message...
2017/06/27 09:21:58 Processed!
2017/06/27 09:21:58 Processed!
2017/06/27 09:21:58 Compressed message
2017/06/27 09:21:58 Processing message...
2017/06/27 09:21:59 Processed!
2017/06/27 09:25:40 Compressed message
2017/06/27 09:25:40 Processing message...
2017/06/27 09:25:41 Compressed message
2017/06/27 09:25:41 Processing message...
2017/06/27 09:25:41 Processed!
2017/06/27 09:25:41 Processed!
2017/06/27 09:25:43 Compressed message
2017/06/27 09:25:43 Processing message...
2017/06/27 09:25:43 Compressed message
2017/06/27 09:25:43 Processing message...
2017/06/27 09:25:43 Processed!
2017/06/27 09:25:43 Processed!
2017/06/27 09:25:43 Compressed message
2017/06/27 09:25:43 Processing message...
2017/06/27 09:25:44 Processed!
2017/06/27 10:36:27 Compressed message
2017/06/27 10:36:27 Processing message...
2017/06/27 10:36:27 Processed!

File: main.go [hunk 1]
```diff
@@ -28,6 +28,10 @@ func main() {
 			Usage: "Location of configuration file",
 		},
 		cli.BoolFlag{
+			Name:  "output, o",
+			Usage: "Logging output from executable",
```
Comment (ricbra) [inline main.go]: I think `"Enable logging of output from executable"` is more clear.

**Review by ricbra — COMMENTED** [review]
Thanks for your contribution. As I will be away a couple of days I'll look into it next week and merge it if all works fine. And release a new version.

**Comment by ricbra** [comment]
Thanks @cravler

**Pull request merged**